public/static/js/subscriptions.js

```diff
@@ -170,6 +170,15 @@ function subCtrl($scope, $mdDialog) {
             $scope.method = t[0];
         }
         $scope.trigger = lookupTrigger($scope.c.triggers, $scope.c.subscription.trigger, $scope.c.subscription.resource_type);
+
+    }else {
+        mciUserSettingsService.getUserSettings({success: function(resp) {
+            if (!$scope.targets[SUBSCRIPTION_SLACK]) {
+                $scope.targets[SUBSCRIPTION_SLACK] = "@" + resp.data.slack_username || "";
+            }
+        }, error: function(resp) {
+            console.log("failed to fetch user settings: ", resp);
```

Comment:
I think you want to log resp.data

Reply:
Nope. It might be correct to log resp.error (since it's an error message), but I want to log the status code too, so I'm logging the whole resp object.